### ISSUE TITLE
Text element inside the script element has zero StreamPosition

### ISSUE THREAD

**Issue by arabcewicz** [zzzprojects/html-agility-pack#175]
Hello,
The following (failing) test illustrates the issue:
```
[Fact]
public void ZeroStreamPositionInsideScript()
{
    var htmlDoc = new HtmlDocument();
    htmlDoc.LoadHtml(@"<html><body><script>foo</script></body></html>");
    var element = htmlDoc.DocumentNode.SelectSingleNode(@"/html[1]/body[1]/script[1]");

    ((HtmlTextNode) element.FirstChild).StreamPosition.Should().BeGreaterThan(0);
}
```
Of course, it should pass.

**Comment by JonathanMagnan** [comment]
Hello @arabcewicz ,

Thank you for reporting.

We will look at your pull.

Best Regards,

Jonathan

**Comment by JonathanMagnan** [comment]
Hello @arabcewicz ,

Thank you for your contribution,

Your pull has been merge & released in v1.8.1

Best Regards,

Jonathan

**Comment by JonathanMagnan** [comment]
Hello @arabcewicz ,

This issue will be closed since it has been resolved.

Feel free to reopen it if you feel otherwise.

Best Regards,

Jonathan.
